includes/REST_API/Products_Controller.php

```diff
@@ -155,7 +155,15 @@ public function get_items( $request ) {
 		$query             = $this->shopping_vendors->get_vendor_class( $shopping_provider );
 
 		if ( ! $query ) {
-			return new WP_Error( 'unable_to_find_class', __( 'Unable to find class', 'web-stories' ), [ 'status' => 400 ] );
+			/** @phpstan-ignore-next-line */
+			$shopping_provider = (string) $this->settings->get_setting( Settings::SETTING_NAME_SHOPPING_PROVIDER );
```

Comment:
```suggestion
```

This is literally done already 5 lines above.

Reply:
removed